src/main/java/nl/workingtalent/bookrental/controller/BookController.java

```diff
@@ -193,14 +160,24 @@ public Book findBook(@PathVariable long id) {
 	}
 	
 	@GetMapping("book/copy/{id}")
```

Comment:
In je verwijderde functie check je wel de token & admin en heb je een error melding. Die moeten wel gewoon blijven lijkt me?